Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/RestrictedActionsApi/RestrictedActionsAPIWrapper.cs

```diff
@@ -61,7 +61,7 @@ public object TriggerSceneEmote(string src, bool loop)
             async UniTask<bool> TriggerSceneEmoteAsync(CancellationToken ct)
             {
                 try { return await api.TryTriggerSceneEmoteAsync(src, loop, ct); }
-                catch (Exception) { return false; }
+                catch (Exception e) when (e is not OperationCanceledException) { return false; }
```

Comment:
Sorry, I missed a word in my previous comment. You should **log** the exception, otherwise it is fully ignored, it's fine to ignore cancellation.

Reply:
Fixed here: https://github.com/decentraland/unity-explorer/pull/1229/commits/d26f03ee70d8688a25a91420ae82affb83b6f454
I've moved the logging into `RestrictedActionsAPIImplementation` since i can add the `SceneShortInfo` into the report